deploy/crds/kieapp.crd.yaml

```diff
@@ -75,58 +75,23 @@ spec:
                       defaultRole:
                         description: A role included for all authenticated users
                         type: string
-                      distinguishedNameAttribute:
-                        description: The name of the attribute in the user entry that
-                          contains the DN of the user. This may be necessary if the
-                          DN of the user itself contains special characters, backslash
-                          for example, that prevent correct user mapping. If the attribute
-                          does not exist, the entry’s DN is used.
+                      newIdentityAttributes:
+                        description: Provide new identities for Ldap  identity mapping,
+                          the pattern to be used with this env is 'attribute_name=attribute_value;another_attribute_name=value'
                         type: string
-                      jaasSecurityDomain:
-                        description: The JMX ObjectName of the JaasSecurityDomain
-                          used to decrypt the password.
-                        type: string
-                      loginModule:
-                        description: A flag to set login module to optional. The default
-                          value is required
-                        enum:
-                        - optional
-                        - required
-                        type: string
```

Comment:
as mentioned in https://github.com/jboss-container-images/rhpam-7-openshift-image/pull/578 - is possible to keep this flag in templates/images and have fallback method for LDAP login module? We should keep this functionality in images if it is possible

Reply:
Not sure if that is possible, since we have migrated the old properties login based to elytron as well. If it is something needed by the user we can try to instruct them to provide a post-configure cli script to do the needed changes, or try to add such option with elytron with aggregate realms.